vendor/ocaml/typing/printtyp.ml

```diff
@@ -1457,6 +1479,69 @@ let unification_error unif tr txt1 ppf txt2 =
     with exn ->
       print_labels := true;
       raise exn
+#else
+
+let super_trace ppf =
+  let rec super_trace first_report ppf = function
+    | (t1, t1') :: (t2, t2') :: rem ->
+      fprintf ppf 
+        "@,@,@[<v 2>";
+      if first_report then 
+        fprintf ppf "The incompatible parts:@,"
+      else begin 
+        fprintf ppf "Further expanded:@,"
+      end;
+      fprintf ppf 
+        "@[<v>\
```

Comment:
It was really hard to read the actual layout of the format otherwise. I could put it on a single line but it'd look like `        "@[<v>@[%a@]@,vs@,@[%a@]%a@]"`